app/assets/javascripts/tictactoe.js

```diff
@@ -1 +1,151 @@
 // Code your JavaScript / jQuery solution here
+
+var turn = 0;
+var currentGame = 0;
+
+WINNING_COMBOS = [
```

Comment:
Don't forget to declare your variables -- since this will never change, you'll probably use `const`.